packages/@storefront/flux-capacitor/src/core/actions/validators.ts

```diff
@@ -72,6 +72,12 @@ export const notOnFirstPage: Validator = {
 
 export const isRefinementDeselectedByIndex: Validator<Actions.Payload.Navigation.Refinement> = {
   func: ({ navigationId, index }, state) => Selectors.isRefinementDeselected(state, navigationId, index),
+  msg: 'navigation does not exist or refinements are already selected'
+};
+
+export const areMultipleRefinementsDeselectedByIndex: Validator<Actions.Payload.Navigation.MultipleRefinements> = {
+  // tslint:disable-next-line max-line-length
+  func: ({ navigationId, indices }, state) => Array.isArray(indices) && indices.some((index) => Selectors.isRefinementDeselected(state, navigationId, index)),
```

Comment:
wait why are you using `some` here and `every` in PP? shouldn't these two behave the exact same?

Reply:
You are right, they should both be the same. This has been updated in the latest commit.